### PR TITLE
Remove usage of deprecated Twig stuff

### PR DESCRIPTION
**Pull request by Koc**

### CODE REVIEW

File: Twig/Extension/FormTypeExtension.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ class FormTypeExtension extends \Twig_Extension
 
     public function __construct($formType)
     {
-        $this->wrapFieldsWithAddons = ($formType == 'standard');
+        $this->wrapFieldsWithAddons = $formType == 'standard';
```
Comment (Soullivaneuh) [inline Twig/Extension/FormTypeExtension.php]: This is not related to the PR. Submit another one with an explanation of why this modification.

**Comment by Koc** [comment]
@Soullivaneuh done

**Comment by Soullivaneuh** [comment]
Thank you @Koc.
